.github/workflows/linter.yml

```diff
@@ -0,0 +1,170 @@
+---
+name: Linter
+
+on:
+  pull_request:
+  workflow_call:
+    inputs:
+      install-all:
+        description: 'Install dependencies using `install-all`'
+        required: false
+        type: boolean
+        default: false
+      generate-keys:
+        description: 'Whether to run command for dummy key generation'
+        required: false
+        type: boolean
+        default: false
+      log-level:
+        description: 'Which log level to use for `super-linter`'
+        required: false
+        type: string
+        default: 'VERBOSE'
+      validate-all-codebase:
+        description: 'Whether to run validation on all files'
+        required: false
+        type: boolean
+        default: true
+
+env:
```

Comment:
very long list - everything necessary? - boilerplate?

Reply:
yes, but this way we only have the list once